docs/installation/base-url.md

```diff
@@ -0,0 +1,46 @@
+# Add ability to run Headlamp with a base-url like "/headlamp"
```

Comment:
Remove the the example from the section title

Reply:
This is done.